premailer/premailer.py

```diff
@@ -235,8 +237,10 @@ def join_css_properties(properties):
                         len(rules)  # this is the rule's index number
                     )
                     rules.append((specificity, selector, bulk))
-
-        return rules, leftover
+        if not self.disable_leftover_css:
+            return rules, leftover
+        else:
+            return rules, None
```

Comment:
This change isn't necessary. Let it return `rules, leftover` here. 
Then [here](https://github.com/ionelmc/premailer/blob/master/premailer/premailer.py#L315) instead you change it to 
```python
if (
    (these_leftover or self.keep_style_tags) and
    not self.disable_leftover_css
):
```
At least I think that would work, but some good new tests would be best to prove it.